### ISSUE TITLE
Conky spamming curl requests with rss

### ISSUE THREAD

**Issue by dimitrissch** [brndnmtthws/conky#779]
My current conky config used to force close after some time running.

After starting conky with -D flag and watching the output it seems to spam the rss servers for data right after it finishes parsing them.This goes on for a while until curl errors (no data received) and conky fcs.

Is something wrong in my configuration?
How can i further debug this?

OS: Arch Linux
Conky version: 1.11.2
Conky config :
```lua
conky.config = {
    background = true,
    update_interval = 5,

    double_buffer = true,
    no_buffers = true,
    text_buffer_size = 4096,
    own_window = true,
    own_window_type = 'normal',
    own_window_class = 'Conky',
    own_window_argb_visual = true,
    own_window_transparent = true,
    own_window_hints = 'undecorated,sticky,skip_taskbar,skip_pager,below',
    alignment = 'bottom_left',
    use_spacer = 'none',
    maximum_width = 550,
    minimum_width = 200,
    max_text_width = 500,
    gap_x = 0,
    gap_y = 20,
    border_inner_margin = 0,
    border_outer_margin = 0,
    border_width = 0,
    draw_shades = false,
    draw_outline = false,
    draw_borders = false,
    draw_graph_borders = false,

    override_utf8_locale = true,
    use_xft = true,
    font = 'Roboto:size=8',
    xftalpha = 0.9,
    uppercase = false,

    default_color = '#FFFFFF',
    color0 = '#9933FF', -- purple
    color1 = '#4CAF50', -- material 500 green
    color2 = '#F44336', -- material 500 red
    color3 = '#F0F8FF', -- whitish
    color4 = '#FFEB3B', -- material 500 yellow

    template0 = [[$color0${font zekton:size=15}\1${color3}\2 $color0${font FontAwesome:size=15}\3]],
};

conky.text = [[
${template0 Calendar Events }
$font$color${texecpi 3600 showscalendar}

${template0 Arch News }
$color$font${rss https://www.archlinux.org/feeds/news/ 30 item_titles 4}

${template0 Deal News }
$color$font${rss https://www.insomnia.gr/forums/forum/56-%CF%80%CF%81%CE%BF%CF%83%CF%86%CE%BF%CF%81%CE%AD%CF%82.xml/ 15 item_titles 10}

${template0 Xda News }
$color$font${rss https://www.xda-developers.com/feed/ 30 item_titles 5}
```

**Comment by lasers** [comment]
>How can i further debug this?

Maybe this will help. Open a terminal and run `gdb conky` then (gdb) `run -c ~/your_conky.conf`. Wait for a crash to occur then do (gdb) `bt full` to get a backtrace. Paste the backtrace here.

>it seems to spam the rss servers for data right after it finishes parsing them.

It seems to be okay on my side. Curls insomnia on 10s interval and others on 30s interval.

**Comment by dimitrissch** [comment]
I misinterpreted the docs and thought the interval was in minutes.Sorry to bother you for something so trivial :(

**Comment by lasers** [comment]
I see the issue. Somebody added http://conky.sourceforge.net/variables.html to the wiki. That link is long updated and hadn't been updated for many years. It did say this for `rss`.

* `rss 	uri interval_in_minutes action (num_par (spaces_in_front))` (sourceforge)
* `rss 	uri interval_in_seconds action (num_par (spaces_in_front))` (man page)

I will remove this reference. Users should use `man -P "less -p 'OBJECTS/VARIABLES'" conky` instead. If users find it a bit hard to parse, try alternatives such as...

* `man -t conky | ps2pdf - conky.pdf` (pdf)
* `man -Hfirefox conky` (firefox)
* http://manpages.ubuntu.com/manpages/latest/en/man1/conky.1.html (ubuntu)

I'll add this in the wiki too. Thanks for the issue report. Cheers. :-)
